components/SearchBar/SearchBar.tsx

```diff
@@ -19,12 +19,18 @@ const SUGGESTED_WORDS_URL = 'https://suggest.finditnowonline.com/SuggestionFeed/
 const SearchBar = ({ big }: SearchProps) => {
   const { t } = useTranslation()
   const router = useRouter()
-  const { query, type, method } = router.query
+
+  // Query params can be of type string[] in case a name is used multiple times in the URL.
+  // We assume here that each name exists only once in the query.
```

Comment:
Do you mean that if i.e. the query is "how to cook" the query will be `['how', 'to', 'cook']`?
If so, why would you assume that each name exist only one?

Reply:
`router.query` is of type `interface ParsedUrlQuery extends NodeJS.Dict<string | string[]> { }`. Meaning that `router.query.query` is of type `string | string[]`. The reason seems to be that the query `query=term+1&type=web` will result in `query` being `"term 1"` while a query `query=term+1&type=web&query=term+2` would result in `["term 1", "term 2"]`.
How can we make this more clear in the comment? Propably a link to some docs?

Reply:
ohh ok i see. Yeah probably an example would help, more or less like you just did here.
But I still don't get it when should be the case that we have two query: `query=term+1&type=web&query=term+2`
in which case this can happen?